CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
+## 3.0.5
+  - Get rid of a Ruby warning from using `timeout`.
+  - Reduce log noise at info level, there shouldn't be a log for every single.
+  - Docs: Better directions for getting a key in Loggly & other cleanups.
+  - Get rid of a few `puts` and reduce the noise at logging level `info`.
+    There is no longer 1+ log/stdout line created for every event ingested.
+
```

Comment:
Nit:  Can you add a reference to the PR that solves each of these (adding #24 to `Get rid of a Ruby warning...`, for example)

Reply:
Good idea

Reply:
So, line 1 is PR #24. All other lines are #26. I'll try a way to format this, let me know your thoughts

Reply:
@robbavey Do you think this looks wonky? https://github.com/webmat/logstash-output-loggly/blob/3468e8f080ca18c74ef92df6c20e6d52bafc69d3/CHANGELOG.md

Is this how it's usually done? Or should I just suck it up and add #26 on each line ;-)

Reply:
@webmat We don't have a standardized changelog format yet. I've typically added the link at the end, although I've not had as many changes rolled into one PR as this one ;)

Reply:
Ok, I thought this looked bad. There's now a full #26 link at the beginning of each line. The code looks repetitive, but the rendered version looks much better.